data/src/main/java/io/quarkus/qe/data/LogEntity.java

```diff
@@ -1,13 +1,12 @@
 package io.quarkus.qe.data;
 
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
 import java.time.LocalDateTime;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
 
-import io.quarkus.hibernate.orm.panache.PanacheEntity;
-
```

Comment:
This change is wrong. The formatter we're using is declaring a different order. Please, undo and make sure you have configured your IDE properly.

Reply:
Still wrong import order.